build.rs

```diff
@@ -24,6 +24,7 @@ fn compress_man() {
         .write(dst, Compression::best());
     let mut ar = tar::Builder::new(encoder);
     ar.mode(tar::HeaderMode::Deterministic);
+    ar.sparse(false);
```

Comment:
This is for internal use only, so may be find with sparse.

Reply:
To be precise, this tarball will be read into the binary at build time, so there shouldn't be any compatibility issue between versions.